pkg/apis/seedmanagement/v1alpha1/defaults_managedseed.go

```diff
@@ -116,7 +116,19 @@ func setDefaultsGardenletConfiguration(obj *gardenletv1alpha1.GardenletConfigura
 
 	// Initialize seed config
 	if obj.SeedConfig == nil {
-		obj.SeedConfig = &gardenletv1alpha1.SeedConfig{}
+		obj.SeedConfig = &gardenletv1alpha1.SeedConfig{
+			SeedTemplate: gardencorev1beta1.SeedTemplate{
+				Spec: gardencorev1beta1.SeedSpec{},
+			},
+		}
```

Comment:
I guess we can revert this change. 

```golang
		obj.SeedConfig = &gardenletv1alpha1.SeedConfig{
			SeedTemplate: gardencorev1beta1.SeedTemplate{
				Spec: gardencorev1beta1.SeedSpec{},
			},
		}
```

should be equivalent to 

```golang
		obj.SeedConfig = &gardenletv1alpha1.SeedConfig{}
```

The `SeedTemplate` and its `Spec` fields are not nil-able. So, we don't have to initialize them to avoid nil pointer.

Or is there any other reason for this change that I am missing?